doc/tutorial.md

```diff
@@ -14,6 +14,10 @@ foodb
 │   └── beforeEach # will be executed before each migration
 ├── grants # use this dir to set special callbacks for grants
 ├── migrations # migrations dir
+│   ├── V0001__Initial_schema_foo.sql    # Migrations should have unique, 
```

Comment:
We probably should drop this change from commit or change commit message.

Reply:
I've split the commit, separating this change into a commit with a more appropriate message